langchain/utils.py

```diff
@@ -19,3 +24,51 @@ def get_from_dict_or_env(
             f" `{env_key}` which contains it, or pass"
             f"  `{key}` as a named parameter."
         )
+
+
+def retry(
```

Comment:
OpenAI documentation also suggests using https://github.com/jd/tenacity , though maybe it is too big for only few things needed.